bitcoin/contrib/test.sh

```diff
@@ -18,6 +18,12 @@ if cargo --version | grep nightly; then
     NIGHTLY=true
 fi
 
+# Pin dependencies as required if we are using MSRV toolchain.
+if cargo --version | grep "1\.41"; then
+    # 1.0.108 uses `matches!` macro so does not work with Rust 1.41.1, bad `syn` no biscuit.
```

Comment:
Double funny because "syn" in Slovak means "son".

Reply:
lol!